src/core/load-fragment.c

```diff
@@ -5581,18 +5581,18 @@ int config_parse_emergency_action(
                 void *data,
                 void *userdata) {
 
-        Manager *m = NULL;
         EmergencyAction *x = ASSERT_PTR(data);
+        Manager *m;
         int r;
 
         assert(filename);
         assert(lvalue);
         assert(rvalue);
 
         if (unit)
-                m = ((Unit*) userdata)->manager;
+                m = ((Unit*) ASSERT_PTR(userdata))->manager;
         else
-                m = data;
+                m = ASSERT_PTR(userdata);
```

Comment:
AFAICS `main()` doesn't pass manager to the config parser (because the parsing is done before the manager is even created), hence this will cause an assert any time `CtrlAltDelBurstAction=` is used in `system.conf`/`user.conf`.

Reply:
you are right.

fuck